salrun/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos:7
+
+LABEL name="LSST SAL Runtime Dependencies" \
+    vendor="LSST" \
+    license="GPLv3"
+
+RUN yum -y install epel-release
+RUN yum -y install python36 python36-devel python36-pip \
+    git make patch wget \
+    gcc-c++
```

Comment:
These are not runtime dependencies.  Including them in every container is excessive.  Note that my versions of the Dockerfiles enabled you to create separate `-build` containers that had the required development tools; you've stripped that out of most of these.